spek-extensions/robolectric/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply from: "$rootDir/gradle/common/dependencies.gradle"
+apply from: "$rootDir/gradle/common/kotlin-jvm.gradle"
+apply from: "$rootDir/gradle/common/junit-jupiter.gradle"
+apply plugin: 'org.jetbrains.dokka'
+
+configurations {
+    provided
+}
+
+sourceSets {
+    main {
+        compileClasspath += configurations.provided
+        runtimeClasspath += configurations.provided
+    }
+    test {
+        compileClasspath += configurations.provided
+        runtimeClasspath += configurations.provided
+    }
+}
+
+dependencies {
+    compile project(':spek-dsl:jvm')
+
+    compile group: 'junit', name: 'junit', version: '4.12'
+    compile group: 'org.robolectric', name: 'robolectric', version: '3.4.2'
+    provided 'org.robolectric:android-all:o-preview-2-robolectric-0'
+    compile 'org.robolectric:android-all:o-preview-2-robolectric-0'
+    testRuntime 'org.robolectric:android-all:o-preview-2-robolectric-0'
```

Comment:
Why do you need same dependency declared as `provided`, `compile` and `testRuntime`? 

`api` should be enough, see https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_configurations_graph

Reply:
Unable to use api in the project. Currently used implementation. Will look into it.